app/controllers/heroku/base_controller.rb

```diff
@@ -1,3 +1,5 @@
+require 'heroku'
```

Comment:
not sure this is a good idea... i will quickly remove all the heroku code as we must do, and then you can just rebase from master once merged :+1: